tracer/src/Datadog.Trace/Agent/Transports/ApiWebRequest.cs

```diff
@@ -62,21 +63,44 @@ public async Task<IApiResponse> PostAsJsonAsync(IEvent events, JsonSerializer se
 
             using (var requestStream = await _request.GetRequestStreamAsync().ConfigureAwait(false))
             {
-                using (var writer = new JsonTextWriter(new StreamWriter(requestStream)))
+                Task WriteStream(Stream stream)
```

Comment:
Should we make this a static local function to avoid the closure?

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1778/commits/62c946f8890c5547eb463e90479a50f783e369bf